invenio_accounts/models.py

```diff
@@ -103,6 +103,9 @@ class User(db.Model, Timestamp, UserMixin):
     _email = db.Column("email", db.String(255), unique=True)
     """User email."""
 
+    domain = db.Column(db.String(255), nullable=True)
```

Comment:
New domain property is added on the user to allow for making a join against the domain table (needed for statistics).